snorkel/labeling/model/label_model.py

```diff
@@ -269,39 +252,13 @@ def predict_proba(self, L):
 
         L_aug = self._get_augmented_label_matrix(L)
```

Comment:
Do we need to update to reflect changed default?

Reply:
just reverted default, I was setting it in the test explicitly